framework/base/Security.php

```diff
@@ -77,8 +77,10 @@ class Security extends Component
      * - 'password_hash' - use of PHP `password_hash()` function with PASSWORD_DEFAULT algorithm.
      *   This option is recommended, but it requires PHP version >= 5.5.0
      * - 'crypt' - use PHP `crypt()` function.
+     * @deprecated Since version 2.0.7, @generatePasswordHash() ignores @passwordHashStrategy and
```

Comment:
I'm not sure, whether `@generatePasswordHash()` is a valid APIDOC reference. Shouldn't it be `[[generatePasswordHash()]]`?